codejail_service/apps/api/v0/views.py

```diff
@@ -143,4 +143,10 @@ def code_exec(request):
         return Response({'globals_dict': globals_out})
     else:
         log.debug("Codejail execution failed for {slug=} with: {error_message}")
+        # Nothing in edxapp actually *uses* the returned globals when there's an
+        # emsg, but it does demand that the key is present in the response. The
+        # globals also aren't updated when codejail encountered an error. We
+        # could just as well return {} here, but we returned the "updated"
+        # globals for compatibility with eduNEXT's existing implementation, just
+        # in case anything actually does care.
```

Comment:
I updated the language to hopefully supply enough information without people needing to know who "we" are, or "eduNEXT", in the future. Thoughts? 
```suggestion
        # globals also aren't updated when codejail encountered an error. We
        # could just as well return {} here, but the service returns the "updated"
        # globals for backward-compatibility, just in case anything actually does
        # care.
```